### ISSUE TITLE
Can't click through to definition of syntax-quoted fn

### ISSUE THREAD

**Issue by tjbrandt** [PEZ/clojure4vscode#21]
I have a fix for this, but I'm logging the issue for tracking purposes.

Not an indentation issue, but cmd-click doesn't take you to the function definition of functions that use the syntax-quote macro (a backtick), which is convention in a certain method of defining routes in the pedestal framework. (See http://pedestal.io/reference/routing-quick-reference#_table_syntax)

In this case, I can cmd-click through to `map` in the first println, but I can't when it's syntax-quoted with a ` in the second println:
```
(println map)
;; => #function[clojure.core/map]
(println `map)
;; => clojure.core/map
```

**Comment by tjbrandt** [comment]
PR is up: https://github.com/PEZ/clojure4vscode/pull/22

**Comment by PEZ** [comment]
This should be fixed now, since #22.
